mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -27,20 +72,48 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 		ARG 2 width
 		ARG 3 height
 		ARG 5 pointer
+	METHOD <init> (Lnet/minecraft/class_1011$class_1012;IIZ)V
+		ARG 2 width
+		ARG 3 height
+	METHOD <init> (Lnet/minecraft/class_1011$class_1012;IIZJ)V
+		ARG 2 width
+		ARG 3 height
+		ARG 5 glPointer
 	METHOD method_15877 writeFile (Ljava/lang/String;)V
-	METHOD method_15990 fromBase64 (Ljava/lang/String;)Lnet/minecraft/class_1011;
+		ARG 1 fileName
+	METHOD method_15990 read (Ljava/lang/String;)Lnet/minecraft/class_1011;
 		ARG 0 string
```

Comment:
In `writeFile`, the string is a file name, and in `read`, the string is the image encoded in base64? This is confusing, either the method name or parameter name should mention it's a base 64 string.

Reply:
Changed to `dataUri` so it's more clear.